_sun.scss

```diff
@@ -1,3 +1,39 @@
+@keyframes bright {
+	0% {
+		transform: translate(-50%, -50%) rotateZ(0);
+	}
+
+	100% {
+		transform: translate(-50%, -50%) rotateZ(360deg);
+	}
+}
+
+@function sunlight($color) {
```

Comment:
What do You think about name "create-sunbeams"? I think it would be more descriptive.

Also - maybe we can use word "sunbeam" instead of "shaft"? (later in the code)